CMake/cdat_modules/x264_deps.cmake

```diff
@@ -1 +1 @@
-set(X264_DEPS "")
+set(X264_deps ${pkgconfig_pkg})
```

Comment:
So X264_deps just depends on pkgconfig? Are there any other system specific libraries we need to document?